<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>vsScaleToolUI</class>
 <widget class="QWidget" name="vsScaleToolUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>869</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scale Tool</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>2</number>
     </property>
     <widget class="QWidget" name="settingsTab">
      <attribute name="title">
       <string>Settings</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="1" column="0" colspan="2">
        <widget class="QGroupBox" name="scaleModelGB">
         <property name="title">
          <string>Scale Model</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="1" colspan="3">
           <widget class="QLineEdit" name="lineEdit"/>
          </item>
          <item row="1" column="0">
           <widget class="QCheckBox" name="checkBox_3">
            <property name="text">
             <string>Marker Data for Mesurements</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="label_8">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>and</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="doubleSpinBox"/>
          </item>
          <item row="0" column="0">
           <widget class="QCheckBox" name="checkBox_2">
            <property name="text">
             <string>Preserve mass distribution during scalling</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Average mesurments between times</string>
            </property>
           </widget>
          </item>
          <item row="0" column="5" rowspan="3">
           <widget class="QGroupBox" name="groupBox_3">
            <property name="title">
             <string>Marker Data</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_5">
             <item row="2" column="0">
              <widget class="QLabel" name="label_11">
               <property name="text">
                <string>Time</string>
               </property>
              </widget>
             </item>
             <item row="2" column="1">
              <widget class="QLabel" name="label_13">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="1" column="1">
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>@</string>
               </property>
              </widget>
             </item>
             <item row="1" column="0">
              <widget class="QLabel" name="label_10">
               <property name="text">
                <string>Frames</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QLabel" name="label_9">
               <property name="text">
                <string>Markers</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_14">
               <property name="text">
                <string>Hz</string>
               </property>
              </widget>
             </item>
             <item row="2" column="2">
              <widget class="QLabel" name="label_15">
               <property name="text">
                <string>s</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="1" column="4">
           <widget class="QToolButton" name="markersMesurmentsTB">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QDoubleSpinBox" name="doubleSpinBox_2"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="2" column="0" colspan="2">
        <widget class="QGroupBox" name="adjustModelMarkersGB">
         <property name="title">
          <string>Adjust Model Markers</string>
         </property>
         <property name="checkable">
          <bool>true</bool>
         </property>
         <layout class="QGridLayout" name="gridLayout_6">
          <item row="0" column="5">
           <widget class="QToolButton" name="markersDataStaticPoseTB">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QDoubleSpinBox" name="markersBetweenTimeFromSB"/>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_18">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="maximumSize">
             <size>
              <width>20</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>and</string>
            </property>
           </widget>
          </item>
          <item row="3" column="2" colspan="3">
           <widget class="QLineEdit" name="coordinateDataStaticPoseLE"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Markers data for static pose</string>
            </property>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QCheckBox" name="previewStaticPoseLE">
            <property name="text">
             <string>Preview static pose (no marker movement)</string>
            </property>
           </widget>
          </item>
          <item row="0" column="2" colspan="3">
           <widget class="QLineEdit" name="markerDataStaticPoseLE"/>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Average markers between times</string>
            </property>
           </widget>
          </item>
          <item row="3" column="5">
           <widget class="QToolButton" name="coordinateDataStaticPoseTB">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QDoubleSpinBox" name="markersBetweenTimesToSB"/>
          </item>
          <item row="3" column="0">
           <widget class="QCheckBox" name="checkBox_4">
            <property name="text">
             <string>Coordinate data for static pose</string>
            </property>
           </widget>
          </item>
          <item row="0" column="6" rowspan="4">
           <widget class="QGroupBox" name="groupBox_4">
            <property name="title">
             <string>Marker Data</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_7">
             <item row="1" column="0" colspan="2">
              <widget class="QLabel" name="label_20">
               <property name="text">
                <string>Frames</string>
               </property>
              </widget>
             </item>
             <item row="3" column="5">
              <widget class="QLabel" name="label_25">
               <property name="text">
                <string>s</string>
               </property>
              </widget>
             </item>
             <item row="1" column="5">
              <widget class="QLabel" name="label_24">
               <property name="text">
                <string>Hz</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0" colspan="2">
              <widget class="QLabel" name="label_19">
               <property name="text">
                <string>Markers</string>
               </property>
              </widget>
             </item>
             <item row="3" column="0">
              <widget class="QLabel" name="label_21">
               <property name="text">
                <string>Time</string>
               </property>
              </widget>
             </item>
             <item row="3" column="2">
              <widget class="QLabel" name="label_23">
               <property name="text">
                <string>-</string>
               </property>
              </widget>
             </item>
             <item row="1" column="2">
              <widget class="QLabel" name="label_22">
               <property name="text">
                <string>@</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Generic Model Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_3">
          <item row="2" column="1">
           <widget class="QLineEdit" name="genericMarkerSetLE"/>
          </item>
          <item row="1" column="1">
           <widget class="QSpinBox" name="genericMassSB"/>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Mass</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>Model Name</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Marker Set</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="genericModelNameLE"/>
          </item>
         </layout>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Subject Data</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="2" column="0">
           <widget class="QCheckBox" name="checkBox">
            <property name="text">
             <string>Add markers from file</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QToolButton" name="markerFileTB">
            <property name="text">
             <string>...</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1" colspan="2">
           <widget class="QDoubleSpinBox" name="massSB"/>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>ModelName</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Mass</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Resulting Marker Set</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="markersFileLE"/>
          </item>
          <item row="3" column="1" colspan="2">
           <widget class="QLineEdit" name="resultingMarkersLE"/>
          </item>
          <item row="0" column="1" colspan="2">
           <widget class="QLineEdit" name="modelNameLE"/>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="scaleFactorsTab">
      <attribute name="title">
       <string>Scale Factors</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_8">
       <item row="1" column="3">
        <widget class="QLineEdit" name="manual2LE"/>
       </item>
       <item row="0" column="0">
        <widget class="QRadioButton" name="measurementRB">
         <property name="text">
          <string>Use Measurements</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="manual1LE"/>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="measurements1CB"/>
       </item>
       <item row="1" column="5">
        <widget class="QLineEdit" name="manual3LE"/>
       </item>
       <item row="0" column="5">
        <widget class="QComboBox" name="mesurement3CB"/>
       </item>
       <item row="1" column="8">
        <widget class="QPushButton" name="resetMesutementBTN">
         <property name="text">
          <string>Reset to Measurement</string>
         </property>
        </widget>
       </item>
       <item row="0" column="8">
        <widget class="QPushButton" name="masmntStBTN">
         <property name="text">
          <string>Edit Measurement Set</string>
         </property>
        </widget>
       </item>
       <item row="0" column="7">
        <widget class="QCheckBox" name="measurmentsUniformCB">
         <property name="text">
          <string>Uniform</string>
         </property>
        </widget>
       </item>
       <item row="0" column="4">
        <widget class="QLabel" name="label_27">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>=</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="label_28">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <widget class="QLabel" name="label_26">
         <property name="maximumSize">
          <size>
           <width>20</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>=</string>
         </property>
         <property name="scaledContents">
          <bool>false</bool>
         </property>
        </widget>
       </item>
       <item row="0" column="3">
        <widget class="QComboBox" name="measurement2CB"/>
       </item>
       <item row="1" column="0">
        <widget class="QRadioButton" name="manualRB">
         <property name="text">
          <string>Use Manual Scales</string>
         </property>
        </widget>
       </item>
       <item row="1" column="7">
        <widget class="QCheckBox" name="manualUniformCB">
         <property name="text">
          <string>Uniform</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="label_29">
         <property name="text">
          <string>=</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0" colspan="9">
        <widget class="QTableView" name="mesurementsTV"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="staticPoseWeightsTab">
      <attribute name="title">
       <string>Static Pose Weights</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Value</string>
           </property>
          </widget>
         </item>
         <item row="0" column="3">
          <widget class="QPlainTextEdit" name="fromFileTE">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="2" column="2">
          <widget class="QRadioButton" name="manualValueRB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Manual Value</string>
           </property>
          </widget>
         </item>
         <item row="1" column="2">
          <widget class="QRadioButton" name="defaultValueRB">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="text">
            <string>Defaul Value</string>
           </property>
          </widget>
         </item>
         <item row="2" column="3">
          <widget class="QTextBrowser" name="manualValueTE">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
           <property name="readOnly">
            <bool>false</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QRadioButton" name="fromFileRB">
           <property name="text">
            <string>From File</string>
           </property>
           <property name="checked">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="disableAllCB">
           <property name="text">
            <string>Disable All Selected</string>
           </property>
          </widget>
         </item>
         <item row="0" column="4">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>weight</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QCheckBox" name="enableAllCB">
           <property name="text">
            <string>Enable All Selected</string>
           </property>
          </widget>
         </item>
         <item row="1" column="3">
          <widget class="QPlainTextEdit" name="defaultValueTE">
           <property name="enabled">
            <bool>false</bool>
           </property>
           <property name="sizePolicy">
            <sizepolicy hsizetype="Maximum" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>30</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="5">
          <widget class="QDoubleSpinBox" name="weightFSP">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="value">
            <double>1.000000000000000</double>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QTableView" name="scaleMarkersTable">
         <property name="selectionMode">
          <enum>QAbstractItemView::ExtendedSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QTableView" name="scaleCoordiantesTable">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="topMargin">
      <number>7</number>
     </property>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="loadBTN">
       <property name="text">
        <string>Load ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="saveBtn">
       <property name="text">
        <string>Save ...</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="runBtn">
       <property name="text">
        <string>Run</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="closeBtn">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="helpBtn">
       <property name="text">
        <string>Help</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
